### ISSUE TITLE
V4.2.2 - Connection Error - websocket disconnected

### ISSUE THREAD

**Issue by lebsd** [OpenEVSE/openevse_esp32_firmware#707]
Firmware 7.1.3 , ESP32r1 2 core WiFi BLE BT, v2 GUI.  

Received multiple times "Connection Error, websocket disconnected, waiting for reconnection".  When this happened on my phone browser, I went to my PC and tried to connect and randomly responded, ie , loaded part of the gui page, then stops, and unresponsive, had to wait a long time and several retries.  This websocket error only started happening with V2 GUI.

Reverted back to 4.1.7 to get control back.

**Comment by lebsd** [comment]
Update 8/5/23:  I have an ethernet cable near the unit from my router.  OpenEvse only has Wifi, To diagnose if this was a connection issue, installed an Access Point 3 ft away from the unit.  Reloaded v2 GUI and V2.2, rebooted the unit.  I was able to connect to the unit's IP from a pc hardwired

24 hrs later, attempted to open the GUI and loaded half way and got stuck for over 1 min, then I was able to get to the network page.  Wifi showed I was connected to the access point but showed only a 1 bar strength even when the access point is 3 ft away.  Attempted to reselect the wifi, but failed.  Restarted the unit, then the wifi went 5 bar strength. Continue monitoring.
